aspnetcore/signalr/streaming/samples/3.0/SignalRStreaming.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="3.0.0-*" />
```

Comment:
I recommend opening an issue for this with a 3.0 label and on the `ASP.NET Core 3.0` project. You can assign to me. Otherwise, I think it will get lost.